Frontend/ui-customer/src/Helpers/Utils.purs

```diff
@@ -504,3 +506,29 @@ fetchDefaultPickupPoint locations lati longi =
   case filter (\loc -> abs(loc.lat - lati) <= 0.0001 && abs(loc.lng - longi) <= 0.0001) locations of
     [foundLocation] -> foundLocation.place
     _ -> ""
+
+getVehicleVariantImage :: String -> String
+getVehicleVariantImage variant =
+  let url = getAssetStoreLink FunctionCall
+  in case getMerchant FunctionCall of
+        YATRISATHI -> case variant of
```

Comment:
remove merchant check from all the places

Reply:
will remove merchant check once all merchant are dependent on common function

Reply:
no need I am working on it